hivemq-edge/src/frontend/src/extensions/datahub/designer/operation/OperationNode.utils.ts

```diff
@@ -76,31 +78,50 @@ export function checkValidityTransformFunction(
   const scriptNodes = functions.map((e) => checkValidityJSScript(e))
   const schemaNodes = serialisers.map((e) => checkValiditySchema(e))
 
-  // TODO[19240] Should not be hardcoded and should be Typescript-ed
+  const scriptName = scriptNodes[0].node as Node<FunctionData>
+  if (!scriptName) {
+    return [
+      {
+        node: operationNode,
+        error: PolicyCheckErrors.notConfigured(operationNode, 'script name'),
+      },
+    ]
+  }
+
+  // TODO[19497] This should not have to happen on the client side!
+  const formattedScriptName = (functionNode: Node<FunctionData>): string => {
+    return `fn:${functionNode.data.name}:${functionNode.data.version}`
+  }
+
   const deserializer: PolicyOperation = {
+    // TODO[19240] Should not be hardcoded and should be Typescript-ed
     functionId: 'Serdes.deserialize',
     arguments: {
       // TODO[19466] Id should come from the node's data when fixed; Need to fix before merging!
       schemaId: serial.source,
-      // TODO[19475] Need to fix before merging!
-      schemaVersion: 1,
-    },
+      // TODO[19497] This is wrong!
+      schemaVersion: '1',
```

Comment:
question(): Since schemaVersion of arguments for Serdes are strings. Why do you think its wrong?